tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestLocale.java

```diff
@@ -238,15 +239,15 @@ public void TestLocalePartsValidity() {
             if (!xpath.startsWith("//ldml/localeDisplayNames/")) {
                 continue;
             }
-            switch (CLDRFile.getNameType(xpath)) {
-                case 0:
```

Comment:
0, 1, and 2 were used here instead of CLDRFile.LOCALE_NAME, CLDRFile.SCRIPT_NAME, and CLDRFile.TERRITORY_NAME